pennylane_lightning/core/src/simulators/lightning_kokkos/measurements/ExpValFunctors.hpp

```diff
@@ -321,7 +330,8 @@ template <class PrecisionT> struct getExpVal2QubitOpFunctor {
     KokkosComplexVector arr;
     KokkosComplexVector matrix;
     const std::size_t n_wires = 2;
-    const std::size_t dim = static_cast<std::size_t>(1U) << n_wires;
+
+    const std::size_t dim = one << n_wires;
```

Comment:
I wonder if we need a convention for naming const values like this?

Reply:
Something like `const_nwires` and `const_dim`?